src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -2046,3 +2047,193 @@ def get_custom_locations_oid(cmd, cl_oid):
+
+        # Adding cli output to the logs
+        diagnostic_checks["diagnoser_results_logger"] = troubleshootutils.cli_output_logger(filepath_with_timestamp, storage_space_available, 1)
+
+        # If all the checks passed then display no error found
+        all_checks_passed = True
+        for checks in diagnostic_checks:
+            if diagnostic_checks[checks] != consts.Diagnostic_Check_Passed:
+                all_checks_passed = False
+
+        if storage_space_available:
+            # Depending on whether all tests passes we will give the output
+            if (all_checks_passed):
+                logger.warning("Diagnoser did not find any issues with the cluster. \nFor more results from the diagnoser refer to the logs collected at " + filepath_with_timestamp + " .\nThese logs can be attached while filing a support ticket for further assistance.\n")
+            else:
+                logger.warning("For more results from the diagnoser refer to the logs collected at " + filepath_with_timestamp + " .\nThese logs can be attached while filing a support ticket for further assistance.\n")
+        else:
+            if (all_checks_passed):
+                logger.warning("Diagnoser did not find any issues with the cluster.\n")
+            logger.warning("Diagnoser was not able to store logs in your local machine. Please check if sufficient storage space is available.\nTo store diagnoser logs clean up some space and execute the troubleshoot command again.")
+
+    # Handling the user manual interrupt
+    except KeyboardInterrupt:
+        try:
+            troubleshootutils.cli_output_logger(filepath_with_timestamp, storage_space_available, 0)
+        except Exception as e:
+            pass
+        raise ManualInterrupt('Process terminated externally.')
+
+
+def install_kubectl_client():
+    # Return kubectl client path set by user
+    try:
+
+        # Fetching the current directory where the cli installs the kubectl executable
+        home_dir = os.path.expanduser('~')
+        operating_system = platform.system().lower()
+
+        # Setting path depending on the OS being used
+        if operating_system == 'windows':
+            kubectl_path = os.path.join(home_dir, '.azure-kubectl', 'kubectl.exe')
+        elif operating_system == 'linux' or operating_system == 'darwin':
+            kubectl_path = os.path.join('usr', 'local', 'bin', 'kubectl')
+
+        # If the kubectl executable already present return the path
+        if os.path.isfile(kubectl_path):
+            return kubectl_path
+
+        # Downloading kubectl executable if its not present in the machine
+        logger.warning("Downloading kubectl client for first time. This can take few minutes...")
+        logging.disable(logging.CRITICAL)
+        get_default_cli().invoke(['aks', 'install-cli'])
+        logging.disable(logging.NOTSET)
```

Comment:
what's this, suppressing output ?

Reply:
check if the above will suppress and you can remove the logging.disable() statements altogether

Reply:
First we suppress the logs and then reset it to default.